### DISCUSSION TITLE
How to calculate surgery duration?

### DISCUSSION THREAD

**Discussion by Mahdi-Dolatkhah** [MIT-LCP/mimic-code#1643] [MIMIC-IV]
Hi,

I want to calculate the surgery duration for each operation. 

I managed to find the start time of the surgery in _poe_detail_ (field_name=Transfer to and field_value=Surgery) and _poe_ (filter based on the poe_id from the previous filtered data). Here, ordertime represent the start time of the surgery. 

Do you have any idea how can I find or calculate the surgery finish time or the duration of the surgery? 

Thanks in advance for your help.

**Comment by alistairewj** [comment]
Unfortunately I don't think that information is captured in MIMIC
currently. There is a separate system for OR data that we haven't
integrated, and we also haven't deidentified the operating notes.

**Reply by Mahdi-Dolatkhah** [reply]
Thank you so much for the information. 

Is there any plan to publish the mentioned data in future? As far as I know, there is no such this information and having them, would be more than beneficial in the field of operating room planning and scheduling. 

Thanks again for your help and support.
